misc/identity-keys/src/identity-keys.ts

```diff
@@ -167,4 +168,8 @@ export class IdentityKeys implements IIdentityKeys {
       throw new Error("Failed to resolve identity key");
     }
   }
+
+  public async getIdentity({ account }: GetIdentityParams): Promise<string> {
+    return this.identityKeys.get(account).identityKeyPub;
```

Comment:
What if there's no identity key for that account ?
Can you add an optional chain operator ?

Reply:
`Store` throws an error if `get` fails, so `?` wouldn't do anything

Source: https://github.com/WalletConnect/walletconnect-monorepo/blob/v2.0/packages/core/src/controllers/store.ts#L147

This is how stores are used throughout repos, with a `try {} catch {}` for `get`.

Reply:
> What if there's no identity key for that account ?

Good call out on this though 👍